oktaawscli/okta_auth.py

```diff
@@ -3,6 +3,7 @@
 import sys
 import time
 import requests
+import logging
```

Comment:
We don't need to import `logging` here, since we're passing the logger in as we instantiate the `OktaAuth` object.

Reply:
sounds good, although general construction/usage of logging is module/class level that I've seen in the wild, the value chain on parameterizing loggers and weaving it through call chains has always felt pretty specialized imo.